deploy-cron.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This is good. But to keep things simple, let's remove this `deploy-cron.sh` file and include the logic in `deploy.sh` instead (in `deploy.sh`, we don't need to copy all of this file in, we just need to change the `gcloud app deploy app-deploy.yaml` line to include `cron.yaml` after `app-deploy.yaml` (space separated list).